platform/vs/docker-sonic-vs/chassis_db.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+import swsssdk
+import json
+chassisdb = swsssdk.SonicV2Connector(host='10.0.0.200', port='6380')
+chassisdb.connect( chassisdb.CHASSIS_DB )
```

Comment:
```suggestion
chassisdb.connect(chassisdb.CHASSIS_DB)
```
I don't think we need the extra spaces (here and there are a few other places as well!).